mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -161,8 +176,9 @@ def __init__(
         req_args = set(config.get("required_args", [])) - set(
             self._tunable_params.get_param_values().keys()
         )
+        req_args.update(self._COMMON_CONST_ARGS)
         merge_parameters(dest=self._const_args, source=global_config, required_keys=req_args)
-        self._const_args = self._expand_vars(self._const_args, global_config or {})
+        self._const_args = self._expand_vars(self._const_args, global_config)
```

Comment:
```suggestion
        self._const_args = self._expand_vars(self._const_args, global_config or {})
```

Tried to merge `main` after the `black` changes, so these should be checked.

Reply:
nm, this was managed above